apps/saleor/package-lock.json

```diff
@@ -19397,6 +19397,18 @@
         "url": "https://github.com/sponsors/sindresorhus"
```

Comment:
definitely not blocking, but this change rolls forward 4 major versions, which makes me think we might need to make more changes in this app?

Reply:
Ok! Will keep an eye on it!